corehq/apps/smsforms/util.py

```diff
@@ -19,6 +19,5 @@ def process_sms_form_complete(session, form, completed):
     session.save()
 
     if not completed:
-        xform.partial_submission = True
-    xform.survey_incentive = session.survey_incentive
```

Comment:
yup, no longer needed - this was a part of some custom work we had done a while back and later removed, and i just missed removing this line